Makefile

```diff
@@ -0,0 +1,13 @@
+all:
+
+clean:
+
+install: all
+	mkdir -p ${DESTDIR}${PREFIX}/bin
+	cp -f vimv ${DESTDIR}${PREFIX}/bin
+	chmod 755 ${DESTDIR}${PREFIX}/bin/vimv
```

Comment:
why not just `install -D -m 0755 vimv ${DESTDIR}${PREFIX}/bin/vimv`
- `-D` will create the directories needed
- `-m` will set the permissions

Reply:
Is `install` POSIX compliant?